### PR TITLE
Removed timeFactor config option and fixed refresh time calculation

### PR DESCRIPTION
**Pull request by Subtletree**
Expiration calculation is now only allowed in seconds, adhering to the
jwt spec.

### CODE REVIEW

File: addon/authenticators/jwt.js [hunk 1]
```diff
@@ -118,14 +109,14 @@ export default TokenAuthenticator.extend({
...
-        expiresAt = this.resolveTime(tokenData[this.tokenExpireName]);
+        expiresAt = tokenData[this.tokenExpireName];
         if (Ember.isEmpty(expiresAt)) {
           return resolve(data);
         }
       }
 
       if (expiresAt > now) {
-        const wait = expiresAt - now - (this.refreshLeeway * 1000);
+        const wait = (expiresAt - now - this.refreshLeeway) * 1000;
```
Comment (jpadilla) [inline addon/authenticators/jwt.js:119]: Why the change in parenthesis order?

**Comment by iamsebastian** [comment]
Due to the fact, that the calculation of how-long-to-wait-before-token-gets-refreshed is imho broken, would be nice, if anybody could merge this.

As this does not makes any sense to me:
https://github.com/jpadilla/ember-simple-auth-token/blob/dd3cda1a6dd6d81a10ed919bf9b6a14fce72af32/addon/authenticators/jwt.js#L200

If I would `hack` the config of my ember-app, and set the `refreshLeeway` to `0.015` for testing out the refresh of the token, the calculated `wait` will force to call the refresh within some milliseconds.

screenshot from 2017-01-17 11-58-50

**Comment by jpadilla** [comment]
@iamsebastian does this PR fix your issue?

**Comment by iamsebastian** [comment]
@jpadilla It looks alike, that this could fix it. (The calculation) Needs to be tested tomorrow - by me. Maybe I install the patch and report back.

**Comment by iamsebastian** [comment]
@jpadilla I've installed the patch via npm/GitHub dependency of @Subtletree 

Looks quite good. Would be great, if you could merge this:
screenshot from 2017-01-17 15-07-06


Thanks in advance

**Review by jpadilla — APPROVED** [review]

**Comment by jpadilla** [comment]
@Subtletree @iamsebastian thanks! Just released this in v2.1.0

File: addon/authenticators/jwt.js (hunk 1 above)
Comment (Subtletree) [reply addon/authenticators/jwt.js:119]: `wait` needs to be in milliseconds for `Ember.run.later`.

Previously `expiresAt` and `now` could be in either seconds or milliseconds which would make the `wait` calculation incorrect if either of them were in seconds.

Now `expiresAt`, `now` and `refreshLeeway` are all in seconds so get combined before being converted to milliseconds.

Comment (iamsebastian) [reply addon/authenticators/jwt.js:119]: @Subtletree is correct here.

**Comment by iamsebastian** [comment]
Thanks a bunch, @Subtletree and @jpadilla
